src/FOMObot/Helpers/Bot.hs

```diff
@@ -27,25 +28,24 @@ getFOMOChannel = do
     channelFinder = find (views Slack.channelName (== "fomo"))
 
 processMessage :: Slack.Event -> Bot Bool
-processMessage (Slack.Message channelID _ _ messageTimestamp _ _) = do
+processMessage (Slack.Message channelID (Slack.UserComment userID) _ messageTimestamp _ _) = do
```

Comment:
Other possibilities are `BotComment` or `System` message ... so this not only gives me access to the userId, it also makes sure we're only considering user messages in our algorithm.